app/validation/validator.py

```diff
@@ -580,15 +620,15 @@ def convert_to_datetime(value):
         return datetime.strptime(value, date_format) if value else None
 
     @staticmethod
-    def _get_all_answer_ids(json_to_validate):
-        answer_ids = []
+    def _get_all_answers(json_to_validate):
+        answers = {}
 
         for section in json_to_validate.get('sections'):
             for group in section.get('groups'):
                 for block in group.get('blocks'):
                     if block.get('questions'):
                         for question in block['questions']:
                             for answer in question.get('answers', []):
-                                answer_ids.append(answer['id'])
+                                answers[answer['id']] = {'group_id': group['id']}
```

Comment:
given we now just call this `_get_all_answers` should we not make this `answers[answer['id']] = answer` ?

Reply:
Answers don't contain a reference to their group

Reply:
then i think the name is more confusing as its not getting answers?